src/vaadin-checkbox-group.html

```diff
@@ -201,6 +201,7 @@
             if (addedCheckboxes.some(checkbox => !checkbox.hasAttribute('value'))) {
               console.warn('Please add value attribute to all checkboxes in checkbox group');
             }
+            this._updateValue(this.value);
```

Comment:
nit: Generally it's discouraged to invoke an observer function directly. Could be handled in the `addedCheckboxes` iteration with something like
```
if (checkbox.checked) {
  this._addCheckboxToValue(checkbox.value);
} else if (this.value.indexOf(checkbox.value) > -1) {
  checkbox.checked = true;
}
```

Reply:
done